tests/boards/ehl_crb/gpio/README.rst

```diff
@@ -0,0 +1,19 @@
+.. _GPIO:
```

Comment:
It would help to make this link name unique & descriptive. Example: https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/master/samples/boards/up_squared/gpio_counter/README.rst